src/aboutus.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="Update your description here" />
+	<title>About Us Page</title>
+</head>
+<body>
+	<div class="topnav">
+		<a href="index.html">home</a>
+		<a href="moodselector.html">mood selector</a>
+		<a href="results.html">your music</a>
+		<a class="active" href="aboutus.html">about us</a>
```

Comment:
For your labels, consider using proper case in the HTML, and using `text-transform: lowercase;` in your CSS to achieve the same style. This is better for accessibility.